src/main.cpp

```diff
@@ -371,18 +371,67 @@ class PointCloudReadModule: public RFModule,
             int width   = abs(bb_bot_right(0) - bb_top_left(0)) + 1;
             int height  = abs(bb_bot_right(1) - bb_top_left(1)) + 1;
 
+            Vector imagePosition(2);
```

Comment:
The naming here is confusing. If the vector `imagePosition` is supposed to contain the center of a bounding box, we might as well call it such.

Reply:
Ok, I can change back the name to `center_bb` if it is better.
It was only to have a name that is similar to the call of the function `retrieveObjectPointCloudFromImagePosition`.

Reply:
Maybe the best course of action is something like

```suggestion
            Vector center_bb(2);
            center_bb(0) = bb_top_left(0) + width/2;
            center_bb1) = bb_top_left(1) + height/2;
            return this->retrieveObjectPointCloudFromImagePosition(objectPointCloud, center_bb);
```

Reply:
Agreed

Reply:
Can you please commit the changes?

Reply:
I committed the changes